test/unit/win_interface_library/bin.c

```diff
@@ -0,0 +1,6 @@
+extern void hello(void);
+
+int main(int argc, char **argv) {
```

Comment:
Let's make this file be C++ (`.cc` extension). For this test it is not significant if this file is C++ or C. But C is handled weird in Bazel (we still link libstdc++ for C-only binaries), so I think seeing C here would surprise the reader. C++ is the boring, default choice.